### ISSUE TITLE
[Suggestion] - Get mobs to spawn in multiple spawn locations not relative to the player locations.

### ISSUE THREAD

**Issue by mibby** [garbagemule/MobArena#260]
I was wondering if it was at all possible to force a specific number of spawn points to be used instead of always using the one spawn point closest to the player. There are a lot of spawn points in my arena so what people do is force mobs to spawn closest to them and then run away. My ideas to combat this issue is a config value to always use X amount of spawn points closest to the player to break down the cluster or use a percentile system to spread mobs out across the entire arena.

**Comment by garbagemule** [comment]
You can use specific spawnpoints for each wave:
https://github.com/garbagemule/MobArena/wiki/Setting-up-the-waves#common-nodes

**Comment by mibby** [comment]
Is there any easy way to set a static spawn point with the built-in setup tools? I find re-setting spawns significantly easy to do in-game when I replace my map design every month - just the spawns are dynamic and try to spawn closest to the player always.

Otherwise, forcing specific spawns is exactly what I needed. :)

**Comment by Sikatsu** [comment]
Good point Mibby, for me it's also spawning at the nearest spawnpoint. I think it has something to do with unloaded chunks @garbagemule

**Comment by mibby** [comment]
My chunks are not unloaded. It just spawns wave mobs on the spawn point nearest to the player, always.

**Comment by Sikatsu** [comment]
My arena is quite big, and if i stand at the middle the mobs will spawn at all spawnpoints instead of the nearest one. If it isn't that, then maybe he should add a node like:

Use-all-spawnpoints: true

**Comment by ct-martin** [comment]
Adding for additional reference, #412
